src/Tribe/PUE/Checker.php

```diff
@@ -493,7 +493,7 @@ private function get_api_message( $info ) {
 		 */
 		public function display_license_error_message() {
 			$plugin_info  = $this->json_error;
-			$license_tab = admin_url( 'wp-admin/edit.php?post_type=tribe_events&page=tribe-common' );
+			$license_tab = admin_url( 'edit.php?page=tribe-common&tab=licenses&post_type=tribe_events' );
```

Comment:
A commit changing this URL hid my previous question. Repeating to ensure it isn't lost: do we always want to set the `post_type` here? Tribe Common may be used without TEC.